Cargo.toml

```diff
@@ -40,7 +40,7 @@ crossbeam-channel = "0.5.8"
 gix-features-for-configuration-only = { package = "gix-features", version = "0.34.0", features = [
```

Comment:
This crate must be updated as well to v0.36.

Reply:
Got it, thanks! IMO that can be a separate PR to re-synchronize these dependencies, but that's up to @o2sh's preferred git history style.

Reply:
Yes, we can update `gix-testtools` in a seperate PR